src/components/Attributes/AttributeRow.tsx

```diff
@@ -43,6 +44,12 @@ const useStyles = makeStyles(
   () => ({
     fileField: {
       float: "right"
+    },
+    pullRight: {
+      "& > *": {
+        float: "right",
+        clear: "right"
+      }
```

Comment:
Can we use flexbox or anything but `float`? Preferably without wildcards in css too